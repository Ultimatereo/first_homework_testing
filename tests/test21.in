5
4 3 1 1 1 0
5 4 0 2 2 0
3 0 4 0 3 2
5 2 1 4 4 4
4 3 5 3 2 5
