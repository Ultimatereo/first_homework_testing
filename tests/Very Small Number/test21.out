-0.1
-0.478571
0.628571
1.27857
-0.0714286
